### DISCUSSION TITLE
GenHTTP Roadmap

### DISCUSSION THREAD

**Discussion by Kaliumhexacyanoferrat** [Kaliumhexacyanoferrat/GenHTTP#567] [Polls]
With GenHTTP 9 the focus of the project has shifted to API development (in contrast to a general purpose web server) with accompanying features such as Websockets or automatically generated Open API definitions.

With this poll I would like to receive some input on the roadmap of the project - so what do you think is currently missing or could be improved when it comes to API development with GenHTTP.

**Comment by justin-caldicott** [comment]
Really pleased to have found your project. We've an app using Nancy which runs on Web / Mac / Windows. With Nancy not supported for a while, your project seems really well positioned to fill the gap. A recently updated, maintained, lightweight no-fuss web server. For this use case, ASP.NET Core + Kestrel being bundled into our apps doesn't seem easy/appealing. Looking forward to taking it for a spin.

**Reply by Kaliumhexacyanoferrat** [reply]
Glad to hear 😊 Please let me know if you encounter any issues (especially on macOS) or have feedback on the developer experience 😉
